label_studio/ml_models/api.py

```diff
@@ -100,6 +99,28 @@ def _is_project_compatible(self, project):
                         if input.get('type', '') == 'Text':
                             return True
         return False
+    def _build_inputs_payload(self, project):
+        parsed_config = project.get_parsed_config()
+        # using this to remove duplicate inputs 
+        input_map = {}
+        for tag in parsed_config:
+            for input in parsed_config[tag].get('inputs',[]):
+                input_type = input.get('type',None)
+                input_name = input.get('value', None)
+                if input_type and input_name and f'{input_type}_{input_name}' not in input_map:
+                    input_map[f'{input_type}_{input_name}'] = {
+                        'type': input_type,
+                        'name': input_name
+                    }
+        return list(input_map.values())
+
+    def _build_classes_payload(self, project):
+        labels, _ = get_all_labels(project.label_config)
+        return sorted(list(set([
```

Comment:
Why do we need this to be sorted?

Reply:
I have seen this practice be used before when response contains list to retain consistent order, so that we do not change the order in every time we provide the response, will confuse user, we can either sort it in backend or frontend would do before displaying it, and it makes testing easier.

Reply:
For testing it doesnt matter we can specify that order does not matter 
For displaying consistent results to the use I agree it could be nice to see labels sorted, but Im also thinking that they might have it in their label config in the order that they want already (might not be alphabetical) and we would return it in the same order to them anyway each time

Reply:
So since I put it in a set the order would change as compared to label config on every call, are we okay with that?

Reply:
Thats true from a quick test it looks like `list(set())` is already sorting it, so the sorted call is redundant anyway, I think its ok not an important detail for now at least. Easy change in the future if we ever need

Reply:
Ah yeah, makes sense. Yeah, I'd prefer if we remove the `sorted` in this case, but it's not a blocker for me

Reply:
Agree we should remove sorted